crates/tiny/config.yml

```diff
@@ -29,6 +29,9 @@ servers:
       #
       # - `pass`: Sends `PASS` IRC command when connecting.
       # - `sasl`: For SASL authentication.
+      #    You can either authenticate using a password or use a certificate
+      #    with SASL EXTERNAL. Choose either `username` and `password` _or_ `pem`
+      #    in the section below.
```

Comment:
Because we're talking about `sasl` section already we should drop the "in the section below" part because it may sound like there are two sections (the `sasl` section + "the section below").

Alternatively we could also move this part right above the `sasl:` line.